src/controllers/shared.controllers/director.correspondence.address.search.controller.ts

```diff
@@ -93,53 +91,62 @@ export const postCorrespondenceAddressLookUp = async (req: Request, res: Respons
       return renderPage(res, req, prepareOfficerFiling, jsValidationErrors, isUpdate, directorName);
     }
 
+    let companyAppointment: CompanyAppointment | undefined = undefined;
+    if (isUpdate) {
+      const companyNumber = urlUtils.getCompanyNumberFromRequestParams(req);
+      const appointmentId = originalOfficerFiling.referenceAppointmentId as string;
+      companyAppointment = await getCompanyAppointmentFullRecord(session, companyNumber, appointmentId);
+      prepareOfficerFiling.correspondenceAddressHasBeenUpdated = checkIsCorrespondenceAddressUpdated(
```

Comment:
Do we want to mark this as an update? Service address will only have the postcode and premises values entered for this page. If a user goes back from here to date details it will presumably show as an update pending and fail validation on submission.

Reply:
Yes, Steer from Richard. Discussed in 3 amigos with Matthew and Declan yesterday.